UnitTests/ViewTests.cs

```diff
@@ -543,5 +543,101 @@ public void CanFocus_Set_Changes_TabIndex_And_TabStop ()
 			Assert.Equal (2, v3.TabIndex);
 			Assert.True (v3.TabStop);
 		}
+
+		[Fact]
+		public void Load_And_Initialize_Event_Comparing_With_Added ()
```

Comment:
This test does not test what happens if a `View` is created AFTER `Run` has been called. Right now, `Initialize` will never fire for such cases, which I think is a problem.

Reply:
I don't think so because the `Initialized `event is only useful at the toplevel load and not if you add views dynamically after the `Run`. In this case you already have all the valid configuration off all the current views and you can deal well with them.

Reply:
Remember that the `Initialized `and `Load `events will always run for each toplevel that will be opening.

Reply:
Pease see this:

https://github.com/migueldeicaza/gui.cs/issues/766#issuecomment-656252989